README.rst

```diff
@@ -1,140 +1,47 @@
 Google API Artifact Manager
 ===========================
 
-Google API Artifact manager (artman) is a set of modules used to automate the
-creation of software artifacts related to APIS defined using `protobuf`_ IDL.
+Google API Artifact Manager (``artman``) is a program used to automate the
+creation of API client libraries.
 
-artman is an extensible framework that is responsible for creating all artifacts
-related to an API including
+In order to be consumed by artman, APIs require:
 
-- distribution packages in all supported programming languages
-- generic documentation websites
-- language-specific documentation websites (javadoc, readthedocs, etc)
+  * A `Protocol Buffers`_ description of the API.
```

Comment:
nit: Shall we say we only support proto3?

Reply:
What happens if you send proto2? Does API Compiler not support it?

Reply:
I will get you to an uncharted area. No one knows what could possibly go wrong when you mix proto2 and proto3 in GRPC world.